code/core/src/manager/globals/exports.ts

```diff
@@ -714,6 +715,7 @@ export default {
     'keyToSymbol',
     'merge',
     'mockChannel',
+    'openInEditor',
```

Comment:
Why is this a new export?

Reply:
I have added it as a new API in case other tools want to use it